### PR TITLE
Preparations for 1.0.2-RC1—incl. release notes.

### PR DESCRIPTION
**Pull request by viktorklang**
@reactive-streams/contributors  This is the preparation for 1.0.2-RC1,

I'd like to issue it before the end of the week, so please provide feedback ASAP if you're interested in vetting the release notes etc.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,107 @@
...
+## Announcement:
+
+This update to `Reactive Streams` brings the following improvements over `1.0.1`.
+
+## Highlights:
+
+- Specification
+  + Glossary term added for `Thread-safe`
+  + No breaking/semantical changes
+  + Rule [clarifications](#specification-clarifications-1.0.2-RC1)
```
Comment (akarnokd) [inline RELEASE-NOTES.md]: The `.` in these links are removed by GitHub's markdown and the URL becomes:

`RELEASE-NOTES.md#specification-clarifications-102-rc1`

Comment (viktorklang) [reply RELEASE-NOTES.md]: Nice catch, thanks @akarnokd!

File: build.gradle [hunk 2]
```diff
@@ -42,7 +42,7 @@ subprojects {
             instructionReplace "Bundle-Vendor", "Reactive Streams SIG"
             instructionReplace "Bundle-Description", "Reactive Streams API"
             instructionReplace "Bundle-DocURL", "http://reactive-streams.org"
-            instructionReplace "Bundle-Version", "1.0.1"
+            instructionReplace "Bundle-Version", "1.0.2-RC1"
```
Comment (akarnokd) [inline build.gradle]: Perhaps `$version`?

Comment (viktorklang) [reply build.gradle]: `instructionReplace "Bundle-Version", $version` ?

Comment (akarnokd) [reply build.gradle]: I think String interpolation should work here (but I can be never sure with Gradle):

`instructionReplace "Bundle-Version", "$version"`

Comment (viktorklang) [reply build.gradle]: For some awkward reason "$version" will generate "1.0.1.RC1" iso "1.0.1-RC1" :S

File: tck/src/test/resources/testng.yaml [hunk 3]
```diff
@@ -6,6 +6,5 @@ tests:
     classes:
       - org.reactivestreams.tck.IdentityProcessorVerificationDelegationTest
       - org.reactivestreams.tck.PublisherVerificationTest
-      - org.reactivestreams.tck.BokenExampleTest
```
Comment (ktoso) [inline tck/src/test/resources/testng.yaml]: yeah thanks for removing

**Pull request merged**